src/editor/object_option.cpp

```diff
@@ -0,0 +1,71 @@
+//  SuperTux
+//  Copyright (C) 2015 Hume2 <teratux.mail@gmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include <string>
+#include <vector>
+
+#include "editor/object_option.hpp"
+
+ObjectOption::ObjectOption(MenuItemKind ip_type, std::string text_, void* ip,
+                           std::string key_, bool visible_, bool allow_empty_) :
+  type(ip_type),
+  text(text_),
+  option(ip),
+  key(key_),
+  visible(visible_),
+  allow_empty(allow_empty_),
+  select()
+{
+  select.clear();
+}
+
+ObjectOption::~ObjectOption() {
+
+}
+
+const std::string
+ObjectOption::to_string() {
+  switch (type) {
+    case MN_TEXTFIELD:
+      return *((std::string*)(option));
+    case MN_NUMFIELD:
+      return std::to_string(*((float*)(option)));
+    case MN_INTFIELD:
+      return std::to_string(*((int*)(option)));
+    case MN_TOGGLE:
+      return (*((bool*)(option))) ? _("true") : _("false");
```

Comment:
I think if we set "true" and "false" to be translatable here, things will go horribly wrong.